ui/dps/rdmty/dps.js

```diff
@@ -42,44 +42,44 @@ var ____Class0=React.Component;for(var ____Class0____Key in ____Class0){if(____C
         return (
             this.props.perSecond === '---' ? null :
             React.createElement("li", {
-                className: 'row ' + this.props.job.toLowerCase() + (this.props.isSelf ? ' self' : ''), 
-                onClick: this.props.onClick}, 
+                className: 'row ' + this.props.job.toLowerCase() + (this.props.isSelf ? ' self' : ''),
+                onClick: this.props.onClick},
```

Comment:
？

Reply:
Why there are many trailing whitespaces in this file...

Reply:
But it is not the point in this PR, recover it.

Reply:
(Thank you!)